recipes/pcl/all/conanfile.py

```diff
@@ -0,0 +1,224 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir, replace_in_file
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class PclConan(ConanFile):
+    name = "pcl"
+    description = "Point Cloud Library"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/PointCloudLibrary/pcl"
+    topics = ("computer-vision", "point-cloud", "pcl", "pointcloud")
```

Comment:
The pcl tag is redundant. Could instead add pcd tag for the suppoeted file format and possibly other formats as well.

Reply:
Done for pcd, for others file extensions, there are many according to https://pointclouds.org/documentation/group__io.html, then I prefer to limit to pcd.